src/patternfly/components/EmptyState/empty-state-icon.hbs

```diff
@@ -1,14 +1,14 @@
-{{#if @partial-block}}
-  <div class="pf-c-empty-state__icon{{#if empty-state-icon--modifier}} {{empty-state-icon--modifier}}{{/if}}"
-    {{#if empty-state-icon--attribute}}
-      {{{empty-state-icon--attribute}}}
-    {{/if}}>
-    {{> @partial-block}}
-  </div>
-{{else}}
-<i class="fas fa-{{#if empty-state-icon--type}} fa-{{empty-state-icon--type}}{{else}}cubes{{/if}} pf-c-empty-state__icon{{#if empty-state-icon--modifier}} {{empty-state-icon--modifier}}{{/if}}"
-  aria-hidden="true"
+<div class="pf-c-empty-state__icon{{#if empty-state-icon--modifier}} {{empty-state-icon--modifier}}{{/if}}"
   {{#if empty-state-icon--attribute}}
     {{{empty-state-icon--attribute}}}
-  {{/if}}></i>
-{{/if}}
+  {{/if}}>
+  {{#if @partial-block}}
+    {{> @partial-block}}
+  {{else}}
+    <i class="fas fa-{{#if empty-state-icon--type}} fa-{{empty-state-icon--type}}{{else}}cubes{{/if}}{{#if empty-state-icon--modifier}} {{empty-state-icon--modifier}}{{/if}}"
+    aria-hidden="true"
+    {{#if empty-state-icon--attribute}}
+      {{{empty-state-icon--attribute}}}
+    {{/if}}></i>
```

Comment:
I don't think we need this, we don't typically pass attrs to FA icons. Tho if you want to keep it, the param name will need to be unique - currently it's the same as the param used for the `.pf-c-empty-state__icon` element.